lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,32 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin-right: $nx-spacing-md;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md 0 ($nx-spacing-md - 2px) 0;
+  text-align: center;
+  min-width: 125px;
```

Comment:
What's the basis for this number?

Reply:
> What's the basis for this number?

It's the width of the underline borders in the Zeplin file

Reply:
> > What's the basis for this number?
> 
> It's the width of the underline borders in the Zeplin file

Per changes in Zeplin the min-width is now 76px: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/66e91ffe5e2498415d620ebac48d13c521b24d0c